Update tutorial for rule suppression examples (#1715)
scalacenter/scalafix: docs/developers/tutorial.md

+ }
```

The next step is to extract the parameter at the index of the argument

```scala
val parameter = method.parameterLists.head(i)
val parameterName = parameter.displayName
Patch.addLeft(t, s"$parameterName = ")
```

We're not done quite yet, because the rule does not respect rule suppression comments like `// scalafix:ok`.  Let's add a test case to reproduce this bug

```scala
complete(false) // scalafix:ok; rule suppression
```

> We can fix this bug by specifying that `Patch.left` should be `atomic`

Reviewer: ```suggestion
We can fix this bug by specifying that `Patch.addLeft` should be `atomic`
```

}.flatten.asPatch
```

> Finally, to report a diagnostic we use `Patch.lint` and add `atomic` to enable rule suppression.

Reviewer: see comment below
```suggestion
Finally, to report a diagnostic we use `Patch.lint` (which supports rule suppression comments out of the box)
```

```scala

> Patch.lint(LiteralArgument(t)).atomic

Reviewer: ```suggestion
Patch.lint(LiteralArgument(t))
```

      .map(newConfig => new NoLiteralArguments(newConfig))
  }
  override def fix(implicit doc: SyntacticDocument): Patch = {
    doc.tree.collect {
        case Term.Apply(_, args) =>
          args.collect {
            case t: Lit if config.isDisabled(t) =>

>               Patch.lint(LiteralArgument(t)).atomic

Reviewer: This was not change in [your PR](https://github.com/scalacenter/scalafix-named-literal-arguments/pull/9). That's because [lint patches are atomic by default](https://github.com/scalacenter/scalafix/blob/7462d5151ab7ed4a2d5b26da978a5d4efd3f73e8/scalafix-core/src/main/scala/scalafix/internal/patch/EscapeHatch.scala#L94-L98). 
```suggestion
              Patch.lint(LiteralArgument(t))
```

      .map(newConfig => new NoLiteralArguments(newConfig))
  }
  override def fix(implicit doc: SyntacticDocument): Patch = {

>     doc.tree.collect {

Reviewer: ```suggestion
    doc.tree
      .collect {
```
